### ISSUE TITLE
Better version check.

### ISSUE THREAD

**Issue by Y-Less** [samp-incognito/samp-streamer-plugin#320]
The plugin gives the error:

```
Include file version (0x291) does not match plugin version (0x282001) (script might need to be recompiled with the latest include file)
```

(or similar).  However, since the data is BCD, it is possible to tell whether the plugin or include is later.  But if you include is newer than your plugin, the message still recommends updating the include, not the plugin.  The `!=` should be `<` and `>` checks together.

**Comment by IstuntmanI** [comment]
The error looks like this right now:
https://github.com/samp-incognito/samp-streamer-plugin/blob/77762f2a37bcd546141602082d9bc9ffe3703192/src/utility/amx.cpp#L76

Which is fine enough.

**Comment by samp-incognito** [comment]
Yeah, I could make that more verbose. I'm not convinced it will help the people who always seem to get stumped on this error, though.
